src/api/assets.go

```diff
@@ -27,14 +27,26 @@ func (info *bindInfo) templateFor(r *http.Request) *templateData {
 		ApiUrl: info.ApiUrl,
 	}
 	addr, _, _ := net.SplitHostPort(r.RemoteAddr)
-	if len(net.ParseIP(addr)) == net.IPv6len {
-		// client is ipv6
+	if info.IP6 != "" && info.IP4 != "" {
```

Comment:
Shouldn't this be caused in https://github.com/facebook/augmented-traffic-control/blob/golang/src/atc_api/main.go#L36 already?